discovery/src/main/resources/reference.conf

```diff
@@ -0,0 +1,10 @@
+######################################################
+# Akka Service Discovery Config                      #
+######################################################
+
+akka.discovery {
+
+  # impl libraries should set this value
+  # impl = ...
```

Comment:
didn't we agree on that we shouldn't do this classpath magic?

Reply:
Right, removing this.